README.md

```diff
@@ -176,6 +176,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+To build an application for macOS, run
+
```

Comment:
yeah, I didn't add this because it was mentioned earlier. This is a note similar to Desktop entry for Linux section